src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -199,8 +199,7 @@ const StudentApplication = ({ userProfile, authentication }) => {
     }
 
     // Check if the selected user is already saved on an application in the database
-    try {
-      let existingAppID = await housing.getCurrentApplicationID(applicant.Profile.AD_Username);
+    housing.getCurrentApplicationID(applicant.Profile.AD_Username).then((existingAppID) => {
```

Comment:
~Just want to make sure, how does the `.then` respond to receiving error codes? Because we expect to receive errors like 403 and 404 in certain cases~
I looked up the documentation, and I believe the behavior here is exactly what I intended

Reply:
I believe the housing service actually catches the 404 and returns null. It would be a good idea to include a `.catch()` statement here if you need to handle any errors (including a 403).